src/entity/Organization.ts

```diff
@@ -34,6 +34,7 @@ export class Organization {
   updatedAt: Date;
 
   @Column({ type: 'text' })
+  @Index('IDX_organization_name_unique', { unique: true })
```

Comment:
Added so that organization name is unique to avoid people adding  well known organizations that were already created.

Still anyone can add any organization so we need to be careful and monitor what gets added. Since we will be doing some manual approval before self serve goes live I am counting on that.